std/zip.d

```diff
@@ -293,6 +293,10 @@ final class ZipArchive
     import std.conv : to;
     import std.datetime.systime : DosFileTime;
 
+    enum endOfCentralDirRecordLength = 22;
+
+    enum endOfCentralDirSignature = cast(ubyte[])"PK\x05\x06";
```

Comment:
This is `public` and breaks the type system (it's mutable).
```suggestion
private static immutable ubyte[] endOfCentraliDirSignature = [ 'P', 'K', 5, 6];
```

Reply:
> This is public and breaks the type system (it's mutable).

It's an `enum`, so I don't think so?

Reply:
It does, because it casts a string literal (which is stored in `.rodata`) to mutable.
```D
class Foo
{
    enum foo = cast(ubyte[])"PK\x05\x06";
}

void main ()
{
    ubyte[] a = Foo.foo;
    a[0] = 5;
}
```

On my machine (OSX):
```console
dmd -run test.d
Error: program killed by signal 10
```